sdk/azcore/runtime/request.go

```diff
@@ -105,9 +102,6 @@ func MarshalAsByteArray(req *policy.Request, v []byte, format Base64Encoding) er
 
 // MarshalAsJSON calls json.Marshal() to get the JSON encoding of v then calls SetBody.
 func MarshalAsJSON(req *policy.Request, v interface{}) error {
-	if omit := os.Getenv("AZURE_SDK_GO_OMIT_READONLY"); omit == "true" {
```

Comment:
Is there any way to ensure that nobody is using the "escape hatch"? I have no idea how widely it was published but it's hard to Bing for so I assume it was mostly an internal detail.

Reply:
It was never publicly documented. We just had it in our back pocket to prescribe on an as-needed basis.  At any rate, the generated code no longer emits the `azure:"ro"` tag on model fields so even if a customer were to set this it would have no effect.